apps/server/src/modules/management/converter/bson.converter.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { Test, TestingModule } from '@nestjs/testing';
-import { ObjectId } from 'bson';
+import { ObjectId } from 'mongodb';
```

Comment:
you should import from mikroORM but then the test doesn't work any more. Should be overworked later